gallery/plot_transforms.py

```diff
@@ -89,3 +89,65 @@ def _plot(img, title, **kwargs):
 # performs random affine transform on an image.
 affined_img = T.RandomAffine(degrees=(30, 70), translate=(0.1, 0.3), scale=(0.5, 0.75))(orig_img)
 plot(affined_img, "Affine transformed image")
+
+####################################
+# RandomCrop
+# ------------
+# The :class:`~torchvision.transforms.RandomCrop` transform
+# (see also :func:`~torchvision.transforms.functional.crop`)
+# performs random crop transform on an image.
+randomcrop_img = T.RandomCrop(size=(128, 128))(orig_img)
+plot(randomcrop_img, "Random Cropped transformed  image")
+
+####################################
+# RandomGrayscale
```

Comment:
Since the `Grayscale` transform is very similar and already documented above, I think we don't need a new section for this one. How about just adding (addition in bold) above:

> (see also to_grayscale **and RandomGrayscale**)

make sure that the links are properly rendered, you'll need to use `` :class:`~torchvision.transforms.RandomGrayScale`  ``

Reply:
Thanks @NicolasHug removed RandomGrayscale